main.css

```diff
@@ -499,6 +594,53 @@ footer {
     column-gap: 24px;
   }
 
+  .close-popup-btn {
+    box-sizing: border-box;
+    position: relative;
+    width: 40px;
+    height: 40px;
+    padding: 14px;
+    display: inline-block;
+    background: #ebecf0;
+    float: right;
+    top: 0;
+    bottom: 0;
+    right: 0;
+    margin-bottom: 20px;
+    grid-column-start: 3;
+  }
+
+  .popUpScreen img {
+    grid-column-end: span 3;
+    max-height: 300px;
+    object-fit: cover;
+  }
+
+  .popUpScreen > ul {
+    grid-column-start: 1;
+    grid-row-start: 5;
+  }
+
+  .popUpScreen > h1 {
+    grid-row-end: span 2;
+  }
+
+  .popUpScreen > p {
+    grid-row-start: 6;
+    grid-column-end: span 3;
+  }
+
+  .popUpScreen {
+    padding: 24px;
+    height: calc(100vh - 244px);
+    width: calc(100vw - 520px);
+    top: 122px;
+    left: 260px;
+    display: grid;
+    grid-template-columns: 60% auto 40px;
+    grid-template-rows: auto auto 28px auto auto;
+  }
+
```

Comment:
There are a few issues with your popup on desktop sizes. Kindly make sure that you follow as much as possible Figma's design 🧐 We could work on the font, text color, and margins of the popup to resemble the design! You are on the right path 😎 One last thing, the color of the x button is too light and loses itself sometimes through the design. Please make the necessary changes in order to meet the project requirements 😉

### Your design 
![image](https://user-images.githubusercontent.com/15807343/151406418-0849a73d-c3a6-4aac-aeb3-373f7572d921.png)

### Figma'
![Captura](https://user-images.githubusercontent.com/15807343/151406451-8782911c-55d6-47b4-ba90-cbc8811aa3be.PNG)
s